src/coffee/directives/api/models/child/base-poly-child-model.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('uiGmapgoogle-maps.directives.api')
     (Builder, gFactory) ->
       class BasePolyChildModel extends Builder
         @include GmapUtil
-        constructor: (@scope, @attrs, @map, @defaults, @model) ->
+        constructor: (@scope, @attrs, @map, @defaults, @model, gObjectChangeCb) ->
```

Comment:
Is this really a gObjectChangeCb? It seems like it calls down into the our "@maybeFit" function.  Unless I'm not following this correctly...  If I am though, calling it a gObject callback is a little misleading to me.